ui/src/components/Form/TextField/index.tsx

```diff
@@ -8,7 +8,7 @@ interface TextFieldProps {
   name: string;
   className?: string;
   placeholder?: string;
-  onBlur?: (e: React.FocusEvent<HTMLInputElement>) => void;
+  isTabIndexEnable?: boolean;
```

Comment:
Rather than doing this, how about we simply accept a `tabIndex` prop and set input's `tabIndex` to whatever is provided? That way we can disable it on some fields by passing `-1` as a value.